extensions/ql-vscode/src/cli.ts

```diff
@@ -482,6 +492,20 @@ export class CodeQLCliServer implements Disposable {
     return await this.runJsonCodeQlCliCommand<QuerySetup>(['resolve', 'library-path'], subcommandArgs, 'Resolving library paths');
   }
 
+  /**
+   * Finds the language that a query is analyzing.
```

Comment:
nit: I don't think _analyzing_ is the right word. But I'm not really sure what the best word is. This suggestion is possibly, slightly better.

```suggestion
   * Finds the language that a query targets.
```